static/app/components/lastCommit.tsx

```diff
@@ -41,6 +41,11 @@ function LastCommit({commit, className}: Props) {
       }
       return `${truncated}\u2026`;
     }
+
+    if (commit.repository !== undefined) {
```

Comment:
if `commit.repository` is undefined, the commit message renders as it did before (not clickable).